Update infores-catalog.yaml with our latest EHRMLA May Treat KG

Qi-ISB opened the pull request:
Add infores:isb-EHRMLA-data, please check if the format satisfy the requirement

Qi-ISB commented:
updated the translator wiki page link to confirm now there is a valid page linked to that url

Qi-ISB commented:
Fixed the knowledge level to the accepted value "predicted'

Apologize for not noticing the required input here is "predicted" which is different than what Plate requires at "prediction"

sierra-moxon commented:
Hi @Qi-ISB - thanks for your help here!   My apologies, I have kicked off the automated tests here again; I do see a failed test.  I also added you to the list of folks with action permission (you may need to accept the GH invite in your mailbox) so that you should automatically see the results of the GH actions when you make changes to the branch now, instead of waiting on me to click the button. :)

Qi-ISB commented:
Hi, Sierra

Thanks for the extremely fast response! I did see the invitation and
accepted it.

Thanks for granting me the action permission. I will check what is the
failed test error message now and try to fix and test it on my end.

I am taking over the translator project from some coworker who recently
left ISB, so this is all brand new translator stuff to me. I will certainly
reach out to you if encounter any issues that I can solve on time by myself.

Thanks,

Qi

On Tue, Aug 13, 2024 at 10:56 AM Sierra Moxon ***@***.***>
wrote:

> Hi @Qi-ISB <https://github.com/Qi-ISB> - thanks for your help here! My
> apologies, I have kicked off the automated tests here again; I do see a
> failed test. I also added you to the list of folks with action permission
> (you may need to accept the GH invite in your mailbox) so that you should
> automatically see the results of the GH actions when you make changes to
> the branch now, instead of waiting on me to click the button. :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/biolink/information-resource-registry/pull/19#issuecomment-2286810833>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWQ2LGUSGHIABDD6X2AR6UTZRJCEJAVCNFSM6AAAAABMJGEKAWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOBWHAYTAOBTGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Qi WEI, PhD*
*Data Scientist, Hadlock Lab*
*he|him*

Qi-ISB commented:
Hi, Sierra

The error that caused the validation script to fail is now related to
another infores:kgea. I don't want to edit other researchers' infores cause
I might break something with my limited knowledge on translator.


>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:393>Checking
> URL: https://archive.translator.ncats.io/
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:394>Response
> status code: 502. Retrying...
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:395>Response
> status code: 502. Retrying...
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:396>Response
> status code: 502. Retrying...
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:397>{'id':
> 'infores:kgea', 'status': 'released', 'name': 'Knowledge Graph Exchange
> Archive', 'xref': ['https://archive.translator.ncats.io/'], 'knowledge
> level': 'curated', 'agent type': 'not_provided'}
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:398>Invalid
> infores URL: for Knowledge Graph Exchange Archive
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:399>
> File
> "/home/runner/work/information-resource-registry/information-resource-registry/src/information_resource_registry/scripts/verify_infores.py",
> line 110, in <module>
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:400>
> InformationResource().validate()
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:401>
> File
> "/home/runner/work/information-resource-registry/information-resource-registry/src/information_resource_registry/scripts/verify_infores.py",
> line 106, in validate
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:402>
> raise ValueError("invalid URL" + infores.get("name") + " for " +
> infores.get("id"))
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:403>ValueError:
> invalid URLKnowledge Graph Exchange Archive for infores:kgea
>

The new infores:isb-EHRMLA-data I added seems to pass the test.

Checking URL:
> https://github.com/NCATSTranslator/Translator-All/wiki/Multiomics-EHRMLA
> -May-Treat-KP
>
> <https://github.com/biolink/information-resource-registry/actions/runs/10326629393/job/28722744622#step:5:166>
> infores:isb-EHRMLA-data has valid URL (xref)
>

Could we work together to solve this issue? I am available now for a quick
zoom meeting or even a phone call to discuss. Or later today at 2:30 pm.

Thanks,

Qi

On Tue, Aug 13, 2024 at 10:56 AM Sierra Moxon ***@***.***>
wrote:

> Hi @Qi-ISB <https://github.com/Qi-ISB> - thanks for your help here! My
> apologies, I have kicked off the automated tests here again; I do see a
> failed test. I also added you to the list of folks with action permission
> (you may need to accept the GH invite in your mailbox) so that you should
> automatically see the results of the GH actions when you make changes to
> the branch now, instead of waiting on me to click the button. :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/biolink/information-resource-registry/pull/19#issuecomment-2286810833>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWQ2LGUSGHIABDD6X2AR6UTZRJCEJAVCNFSM6AAAAABMJGEKAWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDEOBWHAYTAOBTGM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
*Qi WEI, PhD*
*Data Scientist, Hadlock Lab*
*he|him*

Qi-ISB commented:
> I have kicked off the automated tests here again; I do see a failed test (but it is for an unrelated resource, KGEA -- which went offline and should be deprecated in the catalog now -- I'll initiate a PR to clean this up and then you can merge the cleanup into your branch?).

Hi, Sierra

Just looping back to this issue, did you have time to clean up the deprecated KGEA infores entry? I didn't see a newer PR in the repo history.

Thanks,

Qi
